dcr/scenarios/ext-seq-multiple-dependencies/run.py

```diff
@@ -9,5 +11,5 @@
 
     test_orchestrator = TestOrchestrator("ExtSeqDependency-VM", tests=tests)
     test_orchestrator.run_tests()
-    test_orchestrator.generate_report_on_vm("test-result-ext-seq-vm.xml")
+    test_orchestrator.generate_report_on_vm(f"test-result-ext-seq-vm-{socket.gethostname()}.xml")
```

Comment:
Will socket.gethostname() always return a hostname ? Just wondering if this will break if the connection is lost

Reply:
This script is run inside the test VM. This call is equivalent to calling `hostname` command on the terminal. There's no network connection required as such. If we're unable to SSH into the VM at all to execute this script, that's a bigger problem then which is handled differently